modules/nutrients/__init__.py

```diff
@@ -55,14 +57,15 @@ def get_calories(self, ndbno):
 def calories_command(bot, trigger):
     query = trigger.group(2)
     results = NDBSearch.search(query)
-    topfood = results.get('list', {}).get('item')
-    if topfood is not None:
-        topfood = [food for food in topfood if food['group'] not in NDBSearch.exclude_foodgroups][0]
-        foodname = topfood['name']
-        ndbno = topfood['ndbno']
-        calories = NDBSearch.get_calories(ndbno)
-        if calories is not None:
-            bot.reply('"{foodname}" has {calories:.0f} kcal per 100 g. See more results at https://ndb.nal.usda.gov/ndb/search/list?qlookup={query}'.format(foodname=foodname, calories=calories, query=requests.utils.quote(query)))
-            return
+    if 'list' in results.keys():
```

Comment:
Fails if the query fails because it returns `False` but that's an easy fix, we can make it return `{}` instead. So this is fine.